tests/TestCase.php

```diff
@@ -93,6 +96,10 @@ abstract class TestCase extends BaseTestCase
         $configurationDefinition = $container->extend(Configuration::class);
         $configurationDefinition->setConcrete($config);
 
+        // disable cache
+        $cacheDefinition = $container->extend(CacheInterface::class);
+        $cacheDefinition->setConcrete(new Psr16Cache(new FilesystemAdapter('phpinsight', 1, '/dev/null')));
```

Comment:
I would expect something like `sys_get_temp_dir`. I am missing something?

Reply:
In fact, two tests was launched on same directory for test exclude configuration. 
I remove this modification by adding a configuration cache key :+1: